config/database.yml.example

```diff
@@ -1,21 +0,0 @@
-base: &base
-  adapter: postgresql
-  username: snpr
-  password: secret
-  pool: 25
-  
-development:
-  <<: *base
-  database: snpr_development
-
-test:
-  <<: *base
-  database: snpr_test
-
-production:
-  <<: *base
-  database: snpr_production
-
-staging:
-  <<: *base
-  database: snpr_production
```

Comment:
See [comment about the `.env.example` file](https://github.com/openSNP/snpr/pull/333/commits/a5a19eaebb34119b5975463883538c4d964855de#r75593377).